### PR TITLE
fixed null identifier issue

### PR DESCRIPTION
**Pull request by hyi**
This PR fixes the null identifier issue, i.e., replace null identifier with DUMMY:000000; otherwise, icees API will raise exception.

### CODE REVIEW

File: bin/initdb_local.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```
Comment (maximusunc) [inline bin/initdb_local.sh:1]: If this is just a debug script, I think it should be named as such so the next person coming in knows. I would also add a comment or something for what variables that `.env` file should have in it.

File: bin/convert.py [hunk 2]
```diff
@@ -30,7 +30,9 @@
                 print("no biolinkType, default to NameThing")
                 categories = ["biolink:NamedThing"]
             ty = column_mapping["type"]
-            idt = old_identifiers[table].get(column, 'DUMMY:000000')
+            idt = old_identifiers[table].get(column)
```
Comment (maximusunc) [inline bin/convert.py:33]: Just for my knowledge (and a comment could potentially be added here in the code), what values are being returned here that was breaking the old code?

**Review by maximusunc — CHANGES_REQUESTED** [review]

File: bin/convert.py (hunk 2 above)
Comment (hyi) [reply bin/convert.py:33]: @maximusunc The old code did not set ```idt``` to DUMMY when the column exists with a value None which happens to occur for many column variables that don't have a linked identifier. So I updated it to get it first, then check if it is not an empty value or None, which works for PCD and covid instances when I tried to create a mapping file for them. With the old code, linked identifiers are set as None or empty string, which triggered the exception from ICEES API code which does not allow empty identifiers.

**Comment by hyi** [comment]
@maximusunc I have addressed your code review comments. See if the added comments make sense.

**Review by maximusunc — APPROVED** [review]

**Pull request merged**